src/state/experiments.js

```diff
@@ -14,11 +22,26 @@ export const LOAD = 'ca/experiments/LOAD';
 export const LOAD_SUCCESS = 'ca/experiments/LOAD_SUCCESS';
 export const LOAD_ERROR = 'ca/experiments/LOAD_ERROR';
 export const SET = 'ca/experiments/SET';
+export const CREATE = 'ca/experiments/CREATE';
```

Comment:
it's not really wrong, but kind of weird. You can see how it's done in redux usually in official examples: https://github.com/reactjs/redux/tree/master/examples/todomvc/src